woocommerce/woocommerce
# API tests for shipping zones and methods

### All Submissions:


### Changes proposed in this Pull Request:

Closes #30676 and #30677.

### How to test the changes in this Pull Request:

1. Clone this branch and run the setup scripts for the e2e environment.
2. Run the new tests for shipping zones and shipping methods:
```
npm test -- --group=shipping-zones
npm test -- --group=shipping-methods
```
3. You should see the `Shipping zones API tests` and `Shipping methods API tests` pass:
```
 PASS  tests/shipping/shipping-zones.test.js
  Shipping zones API tests
    ✓ can create a shipping zone (243ms)
    ✓ can retrieve a shipping zone (88ms)
    ✓ can list all shipping zones (93ms)
    ✓ can update a shipping zone (89ms)
    ✓ can delete a shipping zone (175ms)

 PASS  tests/shipping/shipping-method.test.js
  Shipping methods API tests
    ✓ can add a 'Flat rate' shipping method (217ms)
    ✓ can add a 'Free shipping' shipping method (181ms)
    ✓ can add a 'Local pickup' shipping method (192ms)
```

### Other information:




### Changelog entry

> Added: Shipping Zones API Tests
> Added: Shipping Methods API Tests

### FOR PR REVIEWER ONLY:
